point0:
  name: "cangku"
  pose:
    x: -0.6
    y: 0.1
    th: 1.571
  setp: 6
  action0:
    fun: 6
    num: 2
    parameter0: 0.0
    parameter1: 0.05
  action1:
    fun: 1
    num: 5
    parameter0: 1.0
    parameter1: 2.0
    parameter2: -115
    parameter3: -31
    parameter4: 230   
  action2:
    fun: 1
    num: 5
    parameter0: 2.0
    parameter1: 2.0
    parameter2: -44
    parameter3: -31
    parameter4: 230    
  action3:
    fun: 1
    num: 5
    parameter0: 3.0
    parameter1: 2.0
    parameter2: -115
    parameter3: 32
    parameter4: 230    
  action4:
    fun: 1
    num: 5
    parameter0: 4.0
    parameter1: 2.0
    parameter2: -44
    parameter3: 31
    parameter4: 230
  action5:
    fun: 3
    num: 2
    parameter0: -0.09
    parameter1: 0.0

point1:
  name: "cnc1"
  pose:
    x: 1.3
    y: -0.4
    th: 1.5707
  setp: 3
  action0:
    fun: 6
    num: 2
    parameter0: 0.0
    parameter1: 0.05
  action1:
    fun: 1
    num: 5
    parameter0: 1.0
    parameter1: 0.0
    parameter2: 270
    parameter3: 0
    parameter4: 200    
  action2:
    fun: 3
    num: 2
    parameter0: -0.09
    parameter1: 0.0

point2:
  name: "cnc2"
  pose:
    x: 2.4
    y: -0.4
    th: 1.571
  setp: 3
  action0:
    fun: 6
    num: 2
    parameter0: 0.0
    parameter1: 0.05
  action1:
    fun: 1
    num: 5
    parameter0: 2.0
    parameter1: 0.0
    parameter2: 270
    parameter3: 0
    parameter4: 200    
  action2:
    fun: 3
    num: 2
    parameter0: -0.09
    parameter1: 0.0


point3:
  name: "cnc1"
  pose:
    x: 1.3
    y: -0.4
    th: 1.5707
  setp: 4
  action0:
    fun: 6
    num: 2
    parameter0: 0.0
    parameter1: 0.05
  action1:
    fun: 1
    num: 5
    parameter0: 1.0
    parameter1: 2.0
    parameter2: -115
    parameter3: -31
    parameter4: 230  
  action2:
    fun: 1
    num: 5
    parameter0: 3.0
    parameter1: 0.0
    parameter2: 270
    parameter3: 0
    parameter4: 200  
  action3:
    fun: 3
    num: 2
    parameter0: -0.09
    parameter1: 0.0


point4:
  name: "cnc2"
  pose:
    x: 2.4
    y: -0.4
    th: 1.571
  setp: 4
  action0:
    fun: 6
    num: 2
    parameter0: 0.0
    parameter1: 0.05
  action1:
    fun: 1
    num: 5
    parameter0: 2.0
    parameter1: 2.0
    parameter2: -44
    parameter3: -31
    parameter4: 230  
  action2:
    fun: 1
    num: 5
    parameter0: 4.0
    parameter1: 0.0
    parameter2: 270
    parameter3: 0
    parameter4: 200  
  action3:
    fun: 3
    num: 2
    parameter0: -0.4
    parameter1: 0.0


point5:
  name: "cnc1"
  pose:
    x: 1.3
    y: -0.4
    th: 1.5707
  setp: 3
  action0:
    fun: 6
    num: 2
    parameter0: 0.0
    parameter1: 0.05
  action1:
    fun: 1
    num: 5
    parameter0: 3.0
    parameter1: 2.0
    parameter2: -115
    parameter3: 31
    parameter4: 230  
  action2:
    fun: 3
    num: 2
    parameter0: -0.4
    parameter1: 0.0

point6:
  name: "cnc2"
  pose:
    x: 2.4
    y: -0.4
    th: 1.571
  setp: 3
  action0:
    fun: 6
    num: 2
    parameter0: 0.0
    parameter1: 0.05
  action1:
    fun: 1
    num: 5
    parameter0: 4.0
    parameter1: 2.0
    parameter2: -44
    parameter3: 31
    parameter4: 230  
  action2:
    fun: 3
    num: 2
    parameter0: -0.4
    parameter1: 0.0
  action3:
    fun: 2
    num: 1
    parameter0: 1.0

point7:
  name: "end product"
  pose:
    x: -0.8
    y: -0.4
    th: -1.571
  setp: 5
  action0:
    fun: 6
    num: 2
    parameter0: 7.0
    parameter1: 0.02   
  action1:
    fun: 1
    num: 5
    parameter0: 3.0
    parameter1: 0.0
    parameter2: 290
    parameter3: -40
    parameter4: 210    
  action2:
    fun: 1
    num: 5
    parameter0: 4.0
    parameter1: 0.0
    parameter2: 238
    parameter3: -40
    parameter4: 210
  action3:
    fun: 3
    num: 2
    parameter0: -0.09
    parameter1: 0.0
  action4:
    fun: 2
    num: 1
    parameter0: 1.0

point8:
  name: "AO"
  pose:
    x: 1.7
    y: -0.85
    th: -1.571
  setp: 3
  action0:
    fun: 6
    num: 2
    parameter0: 7.0
    parameter1: 0.02   
  action1:
    fun: 1
    num: 5
    parameter0: 1.0
    parameter1: 0.0
    parameter2: 290
    parameter3: 40
    parameter4: 210   
  action2:
    fun: 1
    num: 5
    parameter0: 2.0
    parameter1: 0.0
    parameter2: 238
    parameter3: 40
    parameter4: 210    

point9:
  name: "AO"
  pose:
    x: 1.7
    y: -0.85
    th: -1.571
  setp: 5
  action0:
    fun: 6
    num: 2
    parameter0: 7.0
    parameter1: 0.02   
  action1:
    fun: 1
    num: 5
    parameter0: 1.0
    parameter1: 2.0
    parameter2: -115
    parameter3: -31
    parameter4: 230  
  action2:
    fun: 1
    num: 5
    parameter0: 3.0
    parameter1: 0.0
    parameter2: 270
    parameter3: 0
    parameter4: 200 
  action3:
    fun: 1
    num: 5
    parameter0: 2.0
    parameter1: 2.0
    parameter2: -44
    parameter3: -31
    parameter4: 230  
  action4:
    fun: 1
    num: 5
    parameter0: 4.0
    parameter1: 0.0
    parameter2: 270
    parameter3: 0
    parameter4: 200  


point10:
  name: "end product"
  pose:
    x: -0.8
    y: -0.4
    th: -1.571
  setp: 3
  action0:
    fun: 6
    num: 2
    parameter0: 7.0
    parameter1: 0.02   
  action1:
    fun: 1
    num: 5
    parameter0: 1.0
    parameter1: 0.0
    parameter2: 290
    parameter3: 40
    parameter4: 210   
  action2:
    fun: 1
    num: 5
    parameter0: 2.0
    parameter1: 0.0
    parameter2: 238
    parameter3: 40
    parameter4: 210    

